packages/vanilla/src/components/radio/radioButton.stories.js

```diff
@@ -0,0 +1,95 @@
+export default {
+  title: 'Patterns',
+  argTypes: {
+    legend: {
+      name: 'Legend Element',
+      description: 'Represents a caption for the content of its parent `<fieldset>` element.',
+      control: { type: 'text' }
+    },
+    inputDescription: {
+      name: 'Input Description',
+      description: 'Instructions or supplementary information regarding the input element. Placed below the `<fieldset>` element.',
+      control: { type: 'text' }
+    },
+    errorMessage: {
+      name: 'Error Message',
+      description: 'A message in the radio button description that a problem has occurred.',
+      control: { type: 'text' }
+    },
+    formControlName: {
+      name: '`name` Attribute',
+      description: '`name` attributes of the radio buttons. **Only one radio button in a same-named group of radio buttons can be checked at a time.**'
+    },
+    radioBtnObj: {
+      name: 'Radio Button Object',
+      description: 'This object contains the attributes/values for the `<input type="checkbox">` elements in the story. This includes the `id`, `label` and `value` attributes. **Setting a value for the `id` key will set the value for the corresponding `<input type="radio">` `id` attribute and associated `<label>` `labelFor` attribute.**'
+    },
+    displayInline: {
+      name: 'Display Inline',
+      description: 'Displays the radio buttons and labels `inline`',
+      control: { type: 'boolean' }
+    },
+    disabled: {
+      name: 'Disabled',
+      description: 'Disable radio button. **In order to see a "checked" disabled radio button, toggle the Checked story control to `true`**',
+      control: { type: 'boolean' }
+    },
+    checked: {
+      name: 'Checked',
+      description: 'Add the `checked` attribute to the radio button (first choice only)',
+      control: { type: 'boolean' }
+    }
+  },
+  decorators: [
+    (Story, context) => `
+      <fieldset class="cbp-fieldset ${context.args.displayInline ? 'cbp-fieldset--inline' : ''}">
```

Comment:
Same as for checklist, but instead of `role="group"` I believe you can use `role="radiogroup"` here (in place of fieldset/legend).